components/PixTutorial.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <section class="tuto">
+    <h1 class="tuto__title">{{ title }}</h1>
+    <p class="tuto__description">{{ description }}</p>
+    <iframe class="tuto__video" :src="videoEmbedSrc" />
+    <ul
+      v-if="videoDlHref || fichePdfHref || transcriptPdfHref"
+      class="tuto__actions"
+    >
+      <li v-if="videoDlHref" class="tuto-actions__item">
+        <PixButtonLink :href="videoDlHref" target="_blank">
+          Télécharger la video
+        </PixButtonLink>
+      </li>
+      <li v-if="fichePdfHref" class="tuto-actions__item">
+        <PixButtonLink :href="fichePdfHref" target="_blank">
```

Comment:
https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable
```suggestion
        <PixButtonLink :href="fichePdfHref" target="_blank" rel="noopener noreferrer">
```